pkg/gofr/gofr.go

```diff
@@ -339,10 +339,15 @@ func (a *App) PATCH(pattern string, handler Handler) {
 func (a *App) add(method, pattern string, h Handler) {
 	a.httpRegistered = true
 
+	reqTimeout, err := strconv.Atoi(a.Config.Get("REQUEST_TIMEOUT"))
+	if err != nil && a.Config.Get("REQUEST_TIMEOUT") != "" {
+		a.container.Error("invalid value of config REQUEST_TIMEOUT. setting default value to 5 seconds.")
```

Comment:
You aren't setting it to 5 here, am I wrong?

I see you copy pasted existing code, but I'm unsure it worked

Reply:
I corrected the code here and now we are checking the value here but not logging here because `add` is being called multiple times to set the default routes and if we log here we will have multiple logs :

<img width="805" alt="Screenshot 2024-08-08 at 11 37 50 AM" src="https://github.com/user-attachments/assets/24c7c26e-3a35-477e-8366-a4f59c6044b9">